src/transaction.rs

```diff
@@ -487,8 +512,10 @@ mod test {
     #[test]
     fn test_drop_db() {
         let dir = TempDir::new("test").unwrap();
-        let env = Environment::new().set_max_dbs(2)
-                                        .open(dir.path()).unwrap();
+        let env = Environment::new()
```

Comment:
The formatting here before was clearly bad, but I think I'd prefer a 'chop-down' style if rustfmt can support it, e.g.

```
        let env = Environment::new().set_max_dbs(2)
                                    .open(dir.path())
                                    .unwrap();
```